server/sonar-server/src/main/java/org/sonar/server/computation/formula/CoreFormulaRepositoryImpl.java

```diff
@@ -51,12 +52,14 @@
     new SumFormula(CLASSES_KEY),
     new SumFormula(FUNCTIONS_KEY),
     new SumFormula(STATEMENTS_KEY),
-    // new SumFormula(PUBLIC_API_KEY),
-    // new SumFormula(COMMENT_LINES_KEY),
-    // new SumFormula(PUBLIC_UNDOCUMENTED_API_KEY),
     new SumFormula(COMPLEXITY_KEY),
     new SumFormula(COMPLEXITY_IN_CLASSES_KEY),
+    // TODO this formula seems to be useless as this measure seems only required on files
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)